src/adapters/rubicon.js

```diff
@@ -410,6 +415,47 @@ RubiconAdapter.masSizeOrdering = function(sizes) {
     });
 };
 
+/**
+ * syncEmily
+ * @summary A user sync dependency for the Rubicon Project adapter
+ * When enabled, creates an user sync iframe after a delay once the first auction is complete.
+ * Only fires once except that with each winning creative there will be additional, similar calls to the same service.
+ * @example
+ *  // Config example for Rubicon user sync
+ *  $$PREBID_GLOBAL$$.rubiconGlobals = {
+ *    userSync: {
+ *      enabled: true,
+ *      delay: 1000
+ *    }
+ *  }
+ * @return {boolean} Whether or not Emily synced
+ */
+function syncEmily(hasSynced) {
+  // Check that it has not already been triggered - only meant to fire once
+  if (hasSynced) {
+    return false;
```

Comment:
this should return `true` because it has already synced, and otherwise `hasUserSyncFired` on line 373 will alternate back and forth between `true` and `false`.